.github/workflows/checks.yml

```diff
@@ -0,0 +1,18 @@
+name: Checks
+on:
+  - pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install modules
+        run: yarn
+      - name: Run type check
+        run: yarn run check:types
+      - name: Run lint check
+        run: yarn run check:lint
+      - name: Run format check
+        run: yarn run check:format
+      - name: Run tests
+        run: yarn test
```

Comment:
**issue:** It looks like the github pr action fails due to an error closing the test database?  See the details on this PR!

Reply:
huh, weird, it passed when I had pushed it up, checking now